5_databases/week2/More_queries.sql

```diff
@@ -0,0 +1,28 @@
+SELECT * FROM user;
+
+SELECT user.name, user.email, task.title 
+FROM user JOIN user_task on user.id = user_id
+          JOIN task ON task.id= task_id
+WHERE email LIKE '%@spotify.com%';
+
+
+
+SELECT user.name, task.title, status.name 
+FROM user JOIN user_task on user.id= user_id
+          JOIN task ON task.id= task_id
+          JOIN status ON status.id= status_id
+WHERE user.name = "Donald Duck";
+
+
+
+SELECT user.name, task.title, task.created
+FROM user JOIN user_task ON user.id= user_id
+          JOIN task on task.id= task_id
+where user.name = "Maryrose Meadows"
+AND MONTH(created) =9;
+
+
+
+SELECT MONTHNAME(created) AS Months, COUNT(title) AS Number_of_tasks
```

Comment:
Cool with MONTHNAME. Usually you would probably use MONTH and just programmatically add the name to the month number in JavaScript